daml-foundations/daml-tools/language-server-tests/package.json

```diff
@@ -16,17 +16,16 @@
         "mocha": "3.2.0",
         "mocha-jenkins-reporter": "0.3.7",
         "typescript": "3.3.3",
-        "vscode-languageserver-types": "3.2.x",
-        "vscode-languageclient": "3.2.x",
-        "vscode-jsonrpc": "3.2.x"
+        "vscode-languageserver-types": "3.14.x",
+        "vscode-languageclient": "5.2.x",
+        "vscode-jsonrpc": "4.0.x"
     },
     "scripts": {
         "postinstall": "node ./node_modules/vscode/bin/install",
         "compile": "node ./node_modules/typescript/bin/tsc -p ./",
         "build": "node ./node_modules/typescript/bin/tsc -p ./"
     },
     "devDependencies": {
-        "@bazel/bazel": "0.22.0",
```

Comment:
Why is this no longer needed? Or why was it needed before?

Reply:
I don’t know why it was needed before but it seems to work fine without it and it takes ages to download so removing it was faster than waiting for `yarn install` to complete :)